TwitchPlaysAssembly/Src/Commands/GameCommands.cs

```diff
@@ -295,8 +295,16 @@ public static void FindClaimView([Group(1)] string commands, [Group(2)] string q
 
 		var claim = commands.ContainsIgnoreCase("claim");
 		var view = commands.ContainsIgnoreCase("view");
-		if (claim || view)
+		if (claim)
+		{
+			if(modules.Count + TwitchGame.Instance.FindClaimPlayers.Count(user) > TwitchGame.Instance.FindClaimUse) {
+				IRCConnection.SendMessageFormat("@{0}, that goes over your current findclaim limit of {1}. Please claim modules manually.", user, TwitchGame.Instance.FindClaimUse);
+				return;
+			}
```

Comment:
This code does have a problem that it doesn't allow users to partially claim the modules that matched. For example, if I ran a command that matches 5 modules on the bomb but I'm only allowed to claim 3 modules it wouldn't allow me to claim 3 of those 5 modules.

Reply:
This specifically was on purpose. I have to move this part around, but we want to stop users from using lists that are too long by not allowing any of their claims.

Reply:
The initial purpose of the PR was to limit the number of terms that a user could submit at once. I  will be adding that as well, but I will change this so it is a `.Take(allowedAmount)` so that you still get a few.